lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py

```diff
@@ -23,94 +24,143 @@
         default: {
-            promiscuous: False,
+            promiscuous_mode: False,
             forged_transmits: False,
             mac_changes: False,
         }
-    teaming_policy:
+        aliases: ['network_policy', 'security_policy']
+    teaming:
         description:
             - Dictionary which configures the different teaming values for portgroup.
             - 'Valid attributes are:'
-            - '- C(load_balance_policy) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
+            - '- C(load_balancing) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
             - '   - choices: [ loadbalance_ip, loadbalance_srcmac, loadbalance_srcid, loadbalance_loadbased, failover_explicit]'
             - '   - "loadbalance_loadbased" is available from version 2.6 and onwards'
-            - '- C(inbound_policy) (bool): Indicate whether or not the teaming policy is applied to inbound frames as well. (default: False)'
+            - '- C(network_failure_detection) (string): Network failure detection. (default: link_status_only)'
+            - '   - choices: [ link_status_only, beacon_probing ]'
             - '- C(notify_switches) (bool): Indicate whether or not to notify the physical switch if a link fails. (default: True)'
-            - '- C(rolling_order) (bool): Indicate whether or not to use a rolling policy when restoring links. (default: False)'
+            - '- C(failback) (bool): Indicate whether or not to use a failback when restoring links. (default: True)'
+            - '- C(failover_order) (dict): Define uplink failover order. Default uplinks are used for active uplinks if not defined.'
+            - '- C(active_uplinks) (list): List of active uplinks used for load balancing.'
+            - '- C(standby_uplinks) (list): List of standby uplinks used for failover.'
+            - '- All uplinks are used as active uplinks if C(active_uplinks) and C(standby_uplinks) are not defined.'
         required: False
         version_added: '2.5'
         default: {
+            'load_balancing': 'loadbalance_srcid',
+            'network_failure_detection': 'link_status_only',
             'notify_switches': True,
-            'load_balance_policy': 'loadbalance_srcid',
-            'inbound_policy': False,
-            'rolling_order': False
+            'failback': True
         }
-    port_policy:
+        aliases: ['teaming_policy']
+    advanced:
         description:
             - Dictionary which configures the advanced policy settings for the portgroup.
             - 'Valid attributes are:'
             - '- C(block_override) (bool): indicates if the block policy can be changed per port. (default: true)'
-            - '- C(ipfix_override) (bool): indicates if the ipfix policy can be changed per port. (default: false)'
-            - '- C(live_port_move) (bool): indicates if a live port can be moved in or out of the portgroup. (default: false)'
```

Comment:
Both `ipfix_override` and `live_port_move` are still available through the API in vSphere 6.7 and can still serve a purpose, this should not be removed:
https://code.vmware.com/apis/358#/doc/vim.dvs.DistributedVirtualPortgroup.PortgroupPolicy.html#livePortMovingAllowed
https://code.vmware.com/apis/358#/doc/vim.dvs.VmwareDistributedVirtualSwitch.VMwarePortgroupPolicy.html

Suggest adding it back